### DISCUSSION TITLE
Hide cursor during screenshot

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [YaLTeR/niri#555] [Ideas]
The ability to hide your cursor while taking a screenshot would be great :)

**Comment by YaLTeR** [comment]
Good thing I just recently implemented it :p

**Reply by ThatOneCalculator** [reply]
Oh nice, will it be in the docs soon?

**Reply by YaLTeR** [reply]
There's a new help panel in the screenshot UI so there's no docs needed.

**Reply by ThatOneCalculator** [reply]
@YaLTeR just got a chance to switch to the latest git version -- looks good! I'd love to have the option to have cursor off by default though :)
